src/ashley/admin.py

```diff
@@ -7,3 +7,5 @@
 @admin.register(User)
 class UserAdmin(admin.ModelAdmin):
     """Admin class for Ashley's User model."""
+
+    search_fields = ["username"]
```

Comment:
Why not add also email, uid, etc while you're at it

Reply:
You're right.
I just added the same fields than in Django's `UserAdmin` + the one that are specific to Ashley.